src/main/java/edu/cornell/kfs/pdp/batch/service/impl/CuAchAdviceNotificationServiceImpl.java

```diff
@@ -15,38 +15,47 @@
  */
 package edu.cornell.kfs.pdp.batch.service.impl;
 
+import java.io.File;
+import java.util.ArrayList;
 import java.util.HashSet;
 import java.util.Iterator;
 import java.util.List;
 
-import org.kuali.kfs.pdp.PdpConstants;
+import javax.mail.internet.AddressException;
+
 import org.kuali.kfs.pdp.batch.service.AchAdviceNotificationService;
 import org.kuali.kfs.pdp.businessobject.CustomerProfile;
 import org.kuali.kfs.pdp.businessobject.PaymentDetail;
 import org.kuali.kfs.pdp.businessobject.PaymentGroup;
-import org.kuali.kfs.pdp.dataaccess.PaymentGroupDao;
 import org.kuali.kfs.pdp.service.PaymentGroupService;
-import org.kuali.kfs.pdp.service.PdpEmailService;
 import org.kuali.kfs.sys.service.NonTransactional;
 import org.kuali.rice.core.api.datetime.DateTimeService;
+import org.apache.commons.collections.CollectionUtils;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 import org.kuali.kfs.krad.service.BusinessObjectService;
 
 import com.rsmart.kuali.kfs.pdp.service.AchBundlerHelperService;
 
+import edu.cornell.kfs.pdp.batch.PDPBadEmailRecord;
+import edu.cornell.kfs.pdp.batch.service.CuAchAdviceNotificationWrrorReportService;
 import edu.cornell.kfs.pdp.dataaccess.AchBundlerAdviceDao;
 import edu.cornell.kfs.pdp.service.CuPdpEmailService;
 
 /**
  * @see org.kuali.kfs.pdp.batch.service.AchAdviceNotificationService
  */
 public class CuAchAdviceNotificationServiceImpl implements AchAdviceNotificationService {
+    private static final Logger LOG = LogManager.getLogger(CuAchAdviceNotificationWrrorReportServiceImpl.class);

```

Comment:
Is CuAchAdviceNotificationWrrorReportServiceImpl correct? Should it match the current class name?

Reply:
good catch, needs fixed